code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -6,7 +6,7 @@
 	)
 	skinned_type = /obj/item/stack/sheet/animalhide/human
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
-	payday_modifier = 1.1
+	payday_modifier = 1.0 // IRIS EDIT, WAS 1.1
```

Comment:
smh, not setting the payday modifier of humans to 0,9 because they are BORING. How could you